flask_dance/contrib/google.py

```diff
@@ -45,6 +46,8 @@ def make_google_blueprint(
         reprompt_consent (bool): If True, force Google to re-prompt the user
             for their consent, even if the user has already given their
             consent. Defaults to False
+        reprompt_select_account (bool): If True, force Google to re-prompt the select account page,
+         even if there is a single logged-in user. Defaults to False
```

Comment:
Can you please make the indentation here match the indentation for the other parameter docstrings?